nebari/template/stages/07-kubernetes-services/modules/kubernetes/services/argo-workflows/main.tf

```diff
@@ -62,9 +72,6 @@ resource "helm_release" "argo-workflows" {
           "${var.node-group.key}" = var.node-group.value
         }
       }
-      controller = {
```

Comment:
This was recently added by @iameskild. Do you know why this is no longer necessary?

Reply:
The emissary Workflow Executor is the default since version 3.3 of Argo Workflows.  The upgraded helm chart deploys v3.4.4 of Argo Workflows.